sydent/db/valsession.py

```diff
@@ -121,6 +121,64 @@ def getTokenSessionById(self, sid):
 
         return None
 
+    def next_link_differs(self, sid, token, next_link):
+        """Check whether the specified session has already been validated with a
+        next_link provided, and if  next_link value
```

Comment:
This sentence doesn't seem to finish?

Reply:
Ah, yes. That's when I was writing the docstring and noticed a problem with the design, so went off to fix it :D